### PR TITLE
catch ChromeDebugException from dwds

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

This might get thrown into the zone, catch it in the zone handler. I don't know if this is necessary any more but I also don't have time to find out.

### CODE REVIEW

File: packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart [hunk 1]
```diff
@@ -216,6 +217,8 @@ class ResidentWebRunner extends ResidentRunner {
       throwToolExit('Failed to connect to WebSocket.');
     } on BuildException {
       throwToolExit('Failed to build application for the Web.');
+    } on ChromeDebugException catch (err) {
+      throwToolExit(err.toString());
```
Comment (zanderso) [inline packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart]: If there's some advice that we can give along with these `throwToolExit()`s, I think that would be generally helpful even if all we can do is say "File a bug", or "Just try it again".

**Review by zanderso — APPROVED** [review]
lgtm w/ optional suggestion for error messages.

File: packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart]: Gave it another shot

**Review by zanderso — APPROVED** [review]
still lgtm

**Review by yjbanov — APPROVED** [review]

**Pull request merged**